cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -131,22 +144,23 @@
 
   # Mount each volume
   dev_path = [] # device labels
-  dev_uuids = [] # device uuids
 
   vol_array.each_with_index do |volumeid, index|
     dev_path[index] = "/dev/disk/by-ebs-volumeid/#{volumeid}"
-    dev_uuids[index] = get_uuid_for_unmount(shared_dir_array[index])
 
     # Unmount and remove volume from /etc/fstab
-    mount shared_dir_array[index] do
-      device(DelayedEvaluator.new { dev_uuids[index] })
-      fstype(DelayedEvaluator.new { node['cluster']['volume_fs_type'] })
-      device_type :uuid
-      options "_netdev"
-      pass 0
-      action %i(unmount disable)
+    execute 'unmount ebs' do
+      command "umount -fl #{shared_dir_array[index]}"
```

Comment:
The following are shared among EBS/EFS/FSx*:
1. the forced-lazy unmounting command;
2. the `not_if` condition on mount
3. the `only_if` condition on enable action
4. retries, retry_delay, timeout

What about putting this command in common?

Reply:
We may want to have different retry_delay here as we discussed for the EFS case

Reply:
Ok, I'll keep track of the action item for clean coding